sunkit_instruments/goes_xrs/__init__.py

```diff
@@ -1 +1,2 @@
 from .goes_xrs import *  # NOQA
+from .goes_chianti_tem import *
```

Comment:
```suggestion
from .goes_chianti_tem import *  # NOQA
```

Reply:
I am not sure we still need these (the precommit should ignore all init files for any removals).